src/jit.rs

```diff
@@ -1627,7 +1623,11 @@ impl<'a, C: ContextObject> JitCompiler<'a, C> {
                     Argument { index: 1, value: Value::RegisterPlusConstant32(REGISTER_PTR_TO_VM, self.slot_in_vm(RuntimeEnvironmentSlot::MemoryMapping), false) },
                     Argument { index: 0, value: Value::RegisterPlusConstant32(REGISTER_PTR_TO_VM, self.slot_in_vm(RuntimeEnvironmentSlot::ProgramResult), false) },
                 ], None);
-            } else {
+            } else { // AccessType::Store
+                if *access_type == 8 {
+                    // Second half of emit_sanitized_load_immediate()
+                    self.emit_ins(X86Instruction::alu(OperandSize::S64, 0x81, 0, RSP, lower_key, Some(X86IndirectAccess::OffsetIndexShift(-96, RSP, 0))));
```

Comment:
How about adding a comment about the -96 offset? Does it come from `stack_slot_of_value_to_store`?